interface/usergroup/mfa_totp.php

```diff
@@ -126,7 +126,7 @@ function docancel() {
                                 </fieldset>
                                 <div class="form-group clearfix">
                                 <div class="col-sm-12 text-left position-override">
-                                        <button type="button" class="btn btn-secondary btn-save" value="<?php echo xla('Submit'); ?>" onclick="doregister('reg2')"><?php echo xlt('Submit'); ?></button>
+                                        <button type="submit" class="btn btn-secondary btn-save" value="<?php echo xla('Submit'); ?>"><?php echo xlt('Submit'); ?></button>
```

Comment:
`<?php echo xla('Submit'); ?>`

Unrelated to this PR - wonder why original code translates submit value? It's like a variable content to be used internally. It can also be set to 'reg2' eliminating need to hardcode function parameter.

Reply:
I was trying to remove the redundant call to `doregister('reg2')`.  Any future update could change it in the `<form>` tag and not in this `<button>` tag.

Reply:
Sorry late. @mdsupport You're correct concerning translating post variables as confusing because if you're acting on the variable, you must use the translated value. This pattern occurs throughout legacy code where I think we just got overzealous at the time we introduced translations.